hack/tests/e2e-create-prod.sh

```diff
@@ -22,6 +21,4 @@ trap cleanup EXIT
 export RUNNING_UNDER_TEST=false
 export TEST_IN_PRODUCTION=true
 
-make create
```

Comment:
If this test isn't being run against the RealRP, will the resource group needed be successfully created?

Reply:
The ginkgo tests against the real rp already have blocks to ensure resource group and cluster creation e.g.
```
BeforeEach(func() {
	var err error
	cli, err = azure.NewClientFromEnvironment(context.Background(), log.GetTestLogger(), false)
	Expect(err).NotTo(HaveOccurred())

	cfg, err = client.NewConfig(tlog.GetTestLogger())
	Expect(err).NotTo(HaveOccurred())

	// create a new resource group
	err = client.EnsureResourceGroup(cfg)
	Expect(err).ToNot(HaveOccurred())
})
```

Reply:
Ah, okay, I was missing that this script only targets the RealRP.